assets/performanceprofile/scripts/add-offline-cpus.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/bash
+
+set -euo pipefail
+
+disable_cpu="0"
+
+cpus="$(echo "${OFFLINE_CPUS}" | grep -o -E '[0-9]+')"
```

Comment:
please consider using the shell builtins here: https://stackoverflow.com/questions/27702452/loop-through-a-comma-separated-shell-variable
```
for i in ${OFFLINE_CPUS//,/ }; do echo "$i"; done
```

Reply:
done